.pre-commit-config.yaml

```diff
@@ -120,6 +122,7 @@ repos:
       WPS440,
       WPS441,
       WPS453,
+      WPS462,
```

Comment:
@woodruffw I'm not sure if this should be ignored. It kinda makes sense to be that gigantic strings in the middle of code don't really help readability... Maybe I'd go for `#noqa: WPS462` where it's absolutely necessary but not ignoring it globally doesn't seem right to me.

Reply:
I'll give that a try. I was having some trouble getting it to be happy with my heredocs, even with `noqa` applied liberally (and attempting to follow its style guidance).

Reply:
Case in point. The very thing the linting rule is trying to prevent seems to happen here...
FWIW I think `# noqa` should be applied on the line with the closing `"""` to work. Though, it looks like we won't be needing this anymore.

Reply:
Yep, I've dropped that ignore now that all of the here-strings are moved up to the module scope.